sqlx-test/src/lib.rs

```diff
@@ -223,3 +224,31 @@ macro_rules! Postgres_query_for_test_prepared_type {
         "SELECT ({0} is not distinct from $1)::int4, {0}, $2"
     };
 }
+
+/// Global lock that prevents multiple tests in this module to be executed at the same time.
+static GLOBAL_LOCK: AtomicBool = AtomicBool::new(false);
```

Comment:
I'm working on a feature in #2001 which should make this unnecessary.

Reply:
shall I wait for you PR then?

Reply:
I just merged it.

Reply:
done